yarn-project/circuits.js/src/keys/derivation.ts

```diff
@@ -123,7 +123,9 @@ export function deriveKeys(secretKey: Fr) {
   };
 }
 
-export function computeTaggingSecret(knownAddress: CompleteAddress, ivsk: Fq, externalAddress: AztecAddress) {
+// Returns shared tagging secret computed with Diffie-Hellman key exchange. Called super secret because
+// it's the secret from which all the other secrets are derived for a given sender-recipient pair.
+export function computeTaggingSuperSecret(knownAddress: CompleteAddress, ivsk: Fq, externalAddress: AztecAddress) {
```

Comment:
I felt like shared secret is a bit overloaded and calling it tagging_super_secret would make it clearer that it's the "uppermost" secret in the hashing onion. Nico called it like that when he was explaining tagging to me and I though the name is good.

Reply:
I find this even more confusing. What's wrong with `secret` and `siloedSecret`? Super doesn't really convey any meaning.

Reply:
maybe `taggingSecretPoint`? It's a concept we use with addresses and points always have to do with deriving from keys.